crates/host/src/module/wasmer_sys.rs

```diff
@@ -54,6 +54,11 @@ pub(crate) fn make_runtime_engine() -> Engine {
     Engine::headless()
 }
 
+/// Build an interpreter module from wasm bytes.
+pub fn build_module(_wasm: &[u8]) -> Result<Arc<Module>, wasmer::RuntimeError> {
+    unimplemented!("The feature flag 'wasmer_wamr' must be enabled to support building a Module directly. Please use the ModuleCache instead.");
```

Comment:
This is keep a consistent api so we don't need to use conditional feature flags in the consumer. 

I recognize this is silly, since the function below that claims to be for ios only, is actually a general way to build a module. I'd like to wait until we remove the prebuild ios modules fully before removing it.